### DISCUSSION TITLE
Translation not working if using Constants file under pages/ folder

### DISCUSSION THREAD

**Discussion by kaisiant** [vinissimus/next-translate#530] [Q&A]
I am trying the use the constant values from a file, so I import the file directly under the pages folder. Then you will see the translated text is wrong. 

pages/Chart.tsx
```
import { TIME_FRAMES, DATE_FORMAT } from '../../constant'
...
const [selectedIndicator, setSelectedIndicator] = useState<
    string | undefined
>(INDICATORS[0].value)
```
constant.ts
```
export const INDICATORS = [
  {
    label: 'rsi',
    value: 'rsi',
  },
  {
    label: 'stoch',
    value: 'stoch',
  },
...
```
It seems the INDICATORS is undefined and the page is having error and caused the translation failure.

But if I create the components outside pages, it works
1. Create a component under components/CandleStickChart.tsx
2. Import to pages/Chart.tsx, the translation works

Is there anyway to solve it?

**Comment by aralroca** [comment]
No idea. It would be great if you would create a reproducible example so we can test and see what is wrong. Thanks
